examples/explorer/src/components/Commons/CursorPagination/CursorPagination.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Pagination } from 'antd';
+import { TABLE_PAGE_SIZE } from '../../../helpers/constants';
+import './cursorPagination.scss';
+
+// Workaround to make Cursor based pagination work with Ant Table
```

Comment:
This refers to that comparison below?
` if (page < current) { `

Reply:
Yep, but also, to the whole component. i'll change the comment to make it more clear